### DISCUSSION TITLE
Mood board mode

### DISCUSSION THREAD

**Discussion by ShyViolets** [goniszewski/grimoire#79] [Ideas]
Hello,
First, this project is really cool! I wanted to submit the idea for another view mode - mood board. Raindrop has something like this- it basically gives a collage feel - no text just images with the pictures of various sizes for an aesthetic looking feel. (I guess Pinterest before it sucked?)

I imagine it would be good for artists who like to college reference images for a project or someone gathering Images for fashion, etc.

so just wanted to throw that out there. Thanks again!

**Comment by goniszewski** [comment]
Hello @ShyViolets! 

Sorry for late reply, I didn't see your message earlier.

This is a really neat idea and I fully support such an enhancement! Most of the bookmarks should already have a main image, and if some of them does not, you can just add them quickly.

It would be the third display type, minimalistic and simple. Sounds like something I would like to use very much 🔥

**Comment by ShyViolets** [comment]
Glad you like the idea :)

idk if it’ll help but I found an image of what it looks like on raindrop - though obv I understand grimoire has its own style. Just wanted to add an image to go with my original description 

[Google images picture](https://images.app.goo.gl/4taDvAuv9vQU1Jga9)
